ebuild-writing/eapi/text.xml

```diff
@@ -62,8 +62,34 @@ EAPI-conditional code)
 When writing new ebuilds developers can choose whatever EAPI they think
 is the best.  Using the features of the latest EAPI is encouraged.
 </p>
+</body>
+
+<subsection>
+<title>Upgrade path</title>
+<body>
+
+<p>
+Gentoo policy is to support upgrades for installations at least a year old
+with no/little intervention and up to 2 years old with minor intervention. To
```

Comment:
Be consistent, either "one year / two years" or "1 year / 2 years". (I'd prefer the former.)

Reply:
Good point.